src/app/+item-page/simple/metadata-representation-list/metadata-representation-list.component.ts

```diff
@@ -1,29 +1,141 @@
+import { ItemMetadataRepresentation } from '../../../core/shared/metadata-representation/item/item-metadata-representation.model';
 
 @Component({
   selector: 'ds-metadata-representation-list',
   templateUrl: './metadata-representation-list.component.html'
 })
 /**
  * This component is used for displaying metadata
- * It expects a list of MetadataRepresentation objects and a label to put on top of the list
+ * It expects an item and a metadataField to fetch metadata
+ * It expects an itemType to resolve the metadata to a an item
+ * It expects a label to put on top of the list
  */
-export class MetadataRepresentationListComponent {
+export class MetadataRepresentationListComponent implements OnInit {
   /**
-   * A list of metadata-representations to display
+   * The parent of the list of related items to display
    */
-  @Input() representations: MetadataRepresentation[];
+  @Input() parentItem: Item;
+
+  /**
+   * The type of item to create a representation of
+   */
+  @Input() itemType: string;
+
+  /**
+   * The metadata field to use for fetching metadata from the item
+   */
+  @Input() metadataField: string;
 
   /**
    * An i18n label to use as a title for the list
    */
   @Input() label: string;
 
+  /**
+   * The max amount of representations to display
+   * Defaults to 10
+   * The default can optionally be overridden by providing the limit as input to the component
+   */
+  @Input() limit = 10;
+
+  /**
+   * A list of metadata-representations to display
+   */
+  representations$: Observable<MetadataRepresentation[]>;
+
   /**
    * The view-mode we're currently on
    * @type {ElementViewMode}
    */
   viewMode = ItemViewMode.Metadata;
+
+  /**
+   * The originally provided limit
+   * Used for resetting the limit to the original value when collapsing the list
+   */
+  originalLimit: number;
+
+  /**
+   * The total amount of metadata values available
+   */
+  total: number;
+
+  constructor(public relationshipService: RelationshipService) {
+  }
+
+  ngOnInit(): void {
+    this.originalLimit = this.limit;
+    this.setRepresentations();
+  }
+
+  /**
+   * Initialize the metadata representations
+   */
+  setRepresentations() {
+    const metadata = this.parentItem.findMetadataSortedByPlace(this.metadataField);
+    this.total = metadata.length;
+    this.representations$ = this.resolveMetadataRepresentations(metadata);
+  }
+
+  /**
+   * Resolve a list of metadata values to a list of metadata representations
+   * @param metadata
+   */
+  resolveMetadataRepresentations(metadata: MetadataValue[]): Observable<MetadataRepresentation[]> {
+    return observableZip(
+      ...metadata
+        .slice(0, this.limit)
+        .map((metadatum: any) => Object.assign(new MetadataValue(), metadatum))
+        .map((metadatum: MetadataValue) => {
+          if (metadatum.isVirtual) {
+            return this.relationshipService.findById(metadatum.virtualValue).pipe(
```

Comment:
I think the problem is within: `this.relationshipService.findById(metadatum.virtualValue)`
I have a metadatum virtual::21, and the metadatum.virtualValue will be 21 
the response of https://host/server/api/core/relationships/21 is:
```
{
  "id": 21,
  "relationshipTypeId": 0,
  "leftPlace": 1,
  "rightPlace": 0,
  "leftwardValue": null,
  "rightwardValue": null,
  "type": "relationship",
  "_links": {
    "relationshipType": {
      "href": "https://host/server/api/core/relationshiptypes/1"
    },
    "self": {
      "href": "https://host/server/api/core/relationships/21"
    },
    "leftItem": {
      "href": "https://host/server/api/core/items/549aa8db-07ee-4e49-9789-822490089d61"
    },
    "rightItem": {
      "href": "https://host/server/api/core/items/9cc6e4cd-756a-468d-b067-5ceff0df6978"
    }
  },
  "_embedded": {
    "relationshipType": {
      "id": 1,
      "leftwardType": "isAuthorOfPublication",
      "rightwardType": "isPublicationOfAuthor",
      "leftMinCardinality": 0,
      "leftMaxCardinality": null,
      "rightMinCardinality": 0,
      "rightMaxCardinality": null,
      "type": "relationshiptype",
```